charts/kured/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.7.0"
 description: A Helm chart for kured
 name: kured
-version: 2.8.0
+version: 2.8.1
```

Comment:
Could you change this to 2.9.0, as its a feature-addition?

Reply:
I've changed this to `2.9.0` now.